debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java

```diff
@@ -1003,6 +1087,7 @@ protected Duration xminFetchInterval() {
                     SNAPSHOT_MODE,
                     SNAPSHOT_MODE_CLASS,
                     HSTORE_HANDLING_MODE,
+                    BINARY_HANDLING_MODE,
```

Comment:
That should be the only change remaining in this class essentially: i.e. pull up the definition as much as possible, but only expose it for PG for the time being.

Reply:
Not sure if I am properly limiting the exposure to PG only.

Reply:
I *think* this is DONE